includes/block.inc

```diff
@@ -15,7 +15,7 @@
  * @param array $block
  *   Options for the block.
  */
-function cm_tools_block_position($module, $delta, array $block = array(), $all_themes = TRUE) {
+function cm_tools_block_position($module, $delta, array $block = array(), $all_themes = TRUE, $types = array()) {
```

Comment:
This extra argument should probably be documented. And you could be a good boy-scout and document `$all_themes` too please?